tests/fixtures/language_container_fixture.py

```diff
@@ -0,0 +1,111 @@
+import contextlib
+import os
+import subprocess
+import textwrap
+import time
+from pathlib import Path
+
+import pytest
+from exasol_bucketfs_utils_python.bucketfs_factory import BucketFSFactory
+from exasol_script_languages_container_tool.lib.tasks.export.export_info import ExportInfo
+from exasol_script_languages_container_tool.lib.tasks.upload.language_definition import LanguageDefinition
+from pytest_itde.config import TestConfig
+
+from exasol_transformers_extension.deployment import language_container
+from tests.utils.parameters import bucketfs_params
+from tests.utils.revert_language_settings import revert_language_settings
+
+
+@pytest.fixture(scope="session")
+def flavor_path() -> Path:
+    return language_container.find_flavor_path()
+
+
+@pytest.fixture(scope="session")
+def export_slc(flavor_path: Path) -> ExportInfo:
+    language_container.prepare_flavor(flavor_path=flavor_path)
+    export_result = language_container.export(flavor_path=flavor_path)
+    export_info = export_result.export_infos[str(flavor_path)]["release"]
+    return export_info
+
+
+@pytest.fixture(scope="session")
+def upload_slc(itde: TestConfig, flavor_path: Path, export_slc: ExportInfo) -> Path:
+    cleanup_images()
+    container_bucketfs_location = \
+        BucketFSFactory().create_bucketfs_location(
+            url=f"{itde.bucketfs.url}/{bucketfs_params.bucket}/{bucketfs_params.path_in_bucket};{bucketfs_params.name}",
+            user=itde.bucketfs.username,
+            pwd=itde.bucketfs.password)
+    container_file_path = Path(export_slc.cache_file)
+    with open(export_slc.cache_file, "rb") as fileobj:
+        container_bucketfs_location.upload_fileobj_to_bucketfs(
+            fileobj,
+            container_file_path.name)
+    with set_language_alias(flavor_path, itde, container_file_path) as language_alias:
+        wait_for_language_container_ready(itde, language_alias)
+    return container_file_path
+
+
+@pytest.fixture(scope="session")
+def language_alias(itde: TestConfig, flavor_path: Path, upload_slc: Path):
+    with set_language_alias(flavor_path, itde, upload_slc) as language_alias:
+        yield language_alias
+
+
+@contextlib.contextmanager
+def set_language_alias(flavor_path: Path, itde: TestConfig, container_file_path: Path):
+    release_name = container_file_path.with_suffix('').with_suffix('').name
```

Comment:
```suggestion
    release_name = container_file_path.with_suffix('').name
```
❓

Reply:
no the file is tar.gz file and has with that two suffixes which we need to remove.